spot_driver/README.md

```diff
@@ -1,4 +1,4 @@
-# spot_driver_cpp
```

Comment:
Update the description in this README to reflect that the package contains both C++ and Python code